package.json

```diff
@@ -173,5 +173,8 @@
     "macos-alias": "npm:@napi-rs/macos-alias@0.0.4",
     "fs-xattr": "npm:@napi-rs/xattr@latest",
     "@radix-ui/react-dialog": "npm:@radix-ui/react-dialog@latest"
+  },
+  "dependencies": {
+    "react-hot-toast": "^2.4.1"
```

Comment:
please do not introduce another toast library. we already have one in `@affine/component`